doc/tests/protocols/MTP_2023-07-8_Heatmap_V1.md

```diff
@@ -0,0 +1,104 @@
+# Test Protocol Heatmap
```

Comment:
Link test protocol from use case.

Reply:
I don't know what this is supposed to mean. There is no test protocol for the use case in plant layer.
Should I link to the usecase? Something like `../usecases/current/plants_layer`?

Reply:
In the use case (plants layer), there should be a link to this test protocol [doc/tests/protocols/MTP_2023-07-8_Heatmap_V1.md](https://github.com/ElektraInitiative/PermaplanT/pull/566/files/6ca9089da16b8961a3b6d5b1bc43be6fd9944446#diff-3bb78915248327a5a3bd26d3a2a3a998e6e1085c948810391bc28d0c8fe27ba3).